sphinx/source/governance.rst

```diff
@@ -1,16 +1,30 @@
 Governance
 ==========
 
-A trusted set of members is in charge of governing a given CCF network. For transparency and auditability, all governance operations are recorded in plaintext in the ledger.
+A trusted set of members is in charge of governing a given CCF network. For transparency and auditability, all governance operations are recorded in plaintext in the ledger and members are required to sign their JSON-RPC requests.
```

Comment:
I think we can leave out the mention of JSON-RPC, "sign their requests" is just as good, and will last longer.